### PR TITLE
Doc: Hardware setup

### PR DESCRIPTION
**Pull request by DanD1511**
Documentation for Arduino setup

### CODE REVIEW

File: content/en/docs/guides/conections_/index.md
Comment (jadarve) [inline content/en/docs/guides/conections_/index.md]: Not sure "implementation" is the right word here.

File: content/en/docs/guides/conections_/index.md
Comment (jadarve) [inline content/en/docs/guides/conections_/index.md]: Not sure this line is needed.

File: content/en/docs/guides/conections_/index.md
Comment (jadarve) [inline content/en/docs/guides/conections_/index.md]: This line could be improved.

File: content/en/docs/guides/setup/android_start/index.md
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: How?

File: content/en/docs/guides/setup/android_start/index.md
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: `title` and `linktitle` are too different.

File: content/en/docs/guides/setup/android_start/index.md
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: why 5?

File: content/en/docs/guides/setup/android_start/index.md
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: This statement could be improved.

**Review by jadarve — COMMENTED** [review]
We could merge this PR once the suggested changes are included.

We probably need to go through another round of editions once we have this one and #4 merged.

File: content/en/docs/guides/conections_/index.md
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: Solved

File: content/en/docs/guides/conections_/index.md
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: Solved

File: content/en/docs/guides/conections_/index.md
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: Solved

File: content/en/docs/guides/setup/android_start/index.md
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: Solved

File: content/en/docs/guides/setup/android_start/index.md
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: Solved

File: content/en/docs/guides/setup/android_start/index.md
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: Solved

File: content/en/docs/guides/conections_/index.md
Comment (aldajo92) [inline content/en/docs/guides/conections_/index.md]: "The image below shows..." looks better.

File: content/en/docs/guides/setup/android_start/index.md
Comment (aldajo92) [inline content/en/docs/guides/setup/android_start/index.md]: you forgot to mention it is possible to connect to raspberry with the hostname also.

File: content/en/docs/guides/setup/android_start/index.md
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: solved

File: content/en/docs/guides/setup/android_start/index.md
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: solved

File: content/en/docs/guides/conections_/index.md
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: solved

**Review by aldajo92 — APPROVED** [review]

**Pull request merged**
